src/client/chat.js

```diff
@@ -355,32 +380,29 @@ module.exports = function (client, options) {
       return
     }
 
-    if (mcData.supportFeature('useChatSessions')) {
-      let acc = 0
-      const acknowledgements = []
-
-      for (let i = 0; i < client._lastSeenMessages.capacity; i++) {
-        const idx = (client._lastSeenMessages.offset + i) % 20
-        const message = client._lastSeenMessages[idx]
-        if (message) {
-          acc |= 1 << i
-          acknowledgements.push(message.signature)
-          message.pending = false
-        }
-      }
-
-      const bitset = Buffer.allocUnsafe(3)
-      bitset[0] = acc & 0xFF
-      bitset[1] = (acc >> 8) & 0xFF
-      bitset[2] = (acc >> 16) & 0xFF
+    if (message.startsWith('/') && mcData.supportFeature('useChatSessions')) {
```

Comment:
Did you review the code ? Why are there 2 different handlings here for chat_command? This is likely what's failing the mineflayer PR. 
![image](https://github.com/PrismarineJS/node-minecraft-protocol/assets/13713600/51fcbb74-d339-49fb-ad00-cd90e6b84928)

Reply:
I think it was messageCount being set to capacity instead of pending that was causing the other pr to fail

Reply:
The two different handlings are because the first is for <1.19.3 and the second (useChatSessions) is for 1.19.3+

Reply:
Mineflayer is passing with the fix

Reply:
Yeah, ok. I think the readability here is very poor with the current code, as it's not obvious at a glance the two conditions are different. Instead I think this should be restructured like

```js
if (message.startsWith('/')) {
  if (supportFeature(a)) { return ... }
  else { return ... }
}
```

for better clarity

Reply:
Agreed, ok, I'll do that. I wasn't sure if it was preferred to have two separate blocks or nested ones